brdnet/blood_only_analysis.ipynb

```diff
@@ -0,0 +1,3223 @@
GDQoEHs2LEjUP7xxx8TFxeH2Wxm+PDhnDhxIui5Z82aRVRUFDabjaeffhq32x0oq3zeyi+9Xs8DDzwQKN+0aRO9e/fGYrHQp08fvv3222pf3+9//3sURWHTpk2BYyNGjAiq22g0Eh4eXu9r2dCaKjYrVqxAp9MFXaNVq1YFtW3t2rX06tULs9lM+/bt+eSTTwD46quvqsROURQWLVoUeOyCBQvo0KEDVquV+++/n9OnTwfK3nzzTW6++WbCw8OJjY3ld7/7HV7v1f9kSpJUIYQQzZ7Xq7H6gzJ+OujBq+j4RL2b9eogtioJfKoOYZt1AEojbIEjRJNS8S8U5rrMOZUJqlVBMStoRZr/mB6UcMVfhx18eT60cg3TL0LRd5YktaGUlJQwcuRIpkyZQn5+PqdOnWLBggVYrdbAOfn5+ZSWlrJlyxZWrlxJcnJyUB0zZsygtLSU3Nxc+vbty6hRowDIysoiKSmJpUuXkp+fT5cuXRg3blzgccuWLeODDz5g586dHDhwgF27djFv3rxAeWlpaeCruLiYVq1aMWbMGAAOHjzImDFjWLRoEcXFxTzzzDOMHDmSoqKioLZlZ2fzySef0KZNm6DjqampQfWPGjUqUPe1oiljA9C3b9+gazR+/PhAWVpaGs8//zx//etfsdvt7Nixg/j4eAAGDhwY9LgdO3agqiqPPPIIAKtWreIvf/kLaWlpnDlzhhtuuCHouX0+H8nJyRQUFLBt2zbWr1/PggULGvbiXgFJUoUQQjR7Px30kH3Ag8XiT0S9io4spSM71Zs4qMSi6mqoQIifISVMQd8tpOp+pxcyAGEKisG/RypeQAHVqOI7e0GCawVNp+E9KosmAfg0D6edOzjs2Mxp5w58Wt2uS3Z2Nm63m4kTJ6LX6wkNDWXw4MEMHDiwyrndunVjwIAB7N69u9q6DAYDkydP5uTJkxQUFPD+++8zYsQIEhMTCQ0NZe7cuWzfvp2DBw8CkJKSwowZM+jQoQPR0dHMmjWrSpJVadOmTZSWlvLwww8Hfr7jjju4++670el0PPHEE4SHh/Ppp58GPe6pp55i4cKFGAyX6s6Hs2fP8tlnnzF58uQrumY18Xo0juxwkrXZwZEdTryeS413v7ymjE1NZs+ezezZs7nzzjtRVZUbbriBuLi4as9NTk7m7rvvpl27dgD8/e9/54knniAuLg6TycTs2bP54osvyMnxLx740ksvkZCQQEhICO3bt2f8+PFs27btitrVkCRJFUII0ew5HBqKAiGX6AA6exbs9obff0+IpqQoCoZbDZe92zM9HEroo2H+uaaOczfzJvCV+aCscusmwA6o4Ml0+8uuYz7NQ2ZJCln2NRwu20yWfQ2ZJSvqlKh27doVo9HI+PHjWb9+PWfOnLnkufv37+err76ic+fO1ZY7nU6WL19Ou3btaNmyJZmZmfTu3TtQHhkZSbt27cjMzASoifJ7BwAAIABJREFUUh4fH8/x48cpLi6uUndKSgqPPfYYoaH+/a19Ph+aFpz8aZpGRkZG4OcVK1YQHR3NsGHDLnsNVq9eTYcOHejfv/9lz7sSXo/Gtykl7FpjJ2tzGbvW2Pl2RUmdEtWmjA1ARkYG0dHRdO7cmZdeeony8nL/a/R6+f7778nPz6dz5860bduWiRMnUlhYWOV5PR4PK1euJCkpKXDs4thV/nvv3r3Vtv3LL7+kV69el3ztjaVOSeqSJUuqdOcLIYQQ16qoKP+fO1dF9eU+H3zy9/Kr2CIhGp+3wEv5x2VgoeodnwqYQd9Gj6GXAeNwE4YBRkL6GVB0CpRdXBlQBL4yzb8tzXUsr2I3Z10HCFHMGNVwQhQzZ13Z5FWk17qu8PBwtm3bhtFoZNq0abRu3ZqhQ4cG9ai1bt2asLAwevbsSWJiItOmTQuq46233sJms9G+fXv27t3LZ599BviH67ZoEbwgnM1mw263V1tus9kAAuWV8vPzWbduXVBP57Bhw/j666/5/PPPcbvdvPPOOxw9epSyMv8vTkFBAXPmzGHx4sU1XoPk5OSgJKo+ju+uIO+AC6NZwRSuYjQrnMl2cSL9Ev/5X0ZTxmbQoEFkZmaSm5vLhg0b2Lp1KzNnzgQgNzcXt9vNRx99xNatW9m/fz9nzpzh+eefr/IaNm7ciNPpZPTo0YFj9913H8uXL+fAgQOUlZUF5gxXxu5Cb7/9Nnv37uWFF16o9fWrrzolqS+99BJt27Zl3LhxpKWlNXSbhBBCiAbVLlbPXYOMuD3+HtWLKQr8kOXh0CEZyiiaD+9h/z6oqlUF47mDChAGSisFJUTBV6Lh+Jsd54ZyKrY5ce9yoamXSUI9Gr7c63t5X6fX31GjKrqg75XHa6tHjx4kJydz5MgRDh48SEhICI8//nig/PTp05SWlvL+++/zzTff4HAEb5f1/PPPU1RURG5uLps3bw7MTbRYLJSUlASdW1xcHJhTeXF5ZQ/qhXMuwT+HsXPnztx+++2BY927d2flypW8+OKLtGrViq+//prExERiYmIAePHFF5k+fTpt27a97Gvfs2cPGRkZ/OpXv7qia1WT8iIvCqDq/P/RV34vK6rb72xTxSYuLo64uDhUVaVbt27Mnz+fNWvWABAWFgbAM888Q0xMDDabjd/97nds3LixSvtTUlIYN24cJpMpcCwpKYmJEycyfPhw4uLi6NmzJ1arNRC7SitXruT111/nH//4By1btqzT9auPOiWpp06dYtGiRRw8eJDExETi4uKYN28ex48fb+j2CSGEEA3izv5GoqNUtGruv30+/5Dg5HcdfPtd7T9xF+KapCrnF0YyKqDDn6TqAKf/mOcnN95TXv+QvzL8+6pWHe15rj4gzL+I0vXMpPP3OPo0b9D3yuP10aFDB6ZPn15l6KWqqkyYMIGEhATmzJlzRXX16tWLPXv2BH4uLCzk6NGjgaGbF5enp6cTExNTpYcvJSWl2p7Ohx56iIyMDM6ePUtycjI//PADd9xxB+Bf2Gfu3
```

Comment:
Might be good to include AUPRC here too, since (I think) your classes are going to be quite imbalanced sometimes: see, for example, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4349800/.

Sklearn AUPRC implementation: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html